### DISCUSSION TITLE
Can't use network mode : share network

### DISCUSSION THREAD

**Discussion by ivan2375** [utmapp/UTM#5190] [UTM Help (macOS)] [answered]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
It doesn’t work since I bought the UTM until now, and no VM installed successfully,

**Configuration**  
* UTM Version: 4.2.2
* macOS Version: 12.6.4
* Mac Chip (Intel, M1, ...): M1

**Crash log**  
![Screen Shot 2023-03-28 at 15 09 16](https://user-images.githubusercontent.com/31252842/228190106-e30c9aca-c0c1-4466-8693-5a31400447cb.jpg)

**Debug log**  

**Upload VM**  

[debug.log](https://github.com/utmapp/UTM/files/11087622/debug.log)

**Comment by osy** [comment, marked as answer]
Restart your Mac. This is an apple bug.
